README.md

```diff
@@ -1,9 +1,32 @@
-## Glitch Grid
+## Motivation
 
-This is a toy distributed system which we will use to demonstrate how to
-transform your system to run within [Antithesis](https://antithesis.com/). We have the same
-basic system implemented in Go and Python, as well as a language-agnostic
-workload which can be used to test the system.
+Glitch Grid is a toy distributed system for demonstrating how to
+interface your system with the [Antithesis Platform](https://antithesis.com/). This project demonstrates:
+
+* Use of an SDK for [assertion definition](https://antithesis.com/docs/using_antithesis/properties.html)
+   and control flow.
+* Build and, for Go, [source instrumentation](https://antithesis.com/docs/instrumentation/go_instrumentation.html)
+* Triggering tests and displaying results via [CI Integrations](https://antithesis.com/docs/using_antithesis/ci.html)
+
+Why demonstrate on a toy project? In order to have interesting test results, we wanted
+to have a software system that contains bugs! Welcome, GlitchGrid.
```

Comment:
Are we implying that most software systems *don't* have bugs? I don't think this is the real reason, and it implies our software isn't good enough to find bugs in other things. I would say instead that we wanted something simple for others to understand the bugs without a lot of prior knowledge.